### PR TITLE
Fix duplicate check

### PR DESCRIPTION
**Pull request by danez**
Fix comment in #107

@kaicataldo Can you check if this is correct?

### CODE REVIEW

File: src/tokenizer/state.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ export default class State {
 
   // Names of exports store. `default` is stored as a name for both
   // `export default foo;` and `export { foo as default };`.
-  exportedIdentifiers: {[id:string]: boolean};
+  exportedIdentifiers: Array<string;
```
Comment (hzoo) [inline src/tokenizer/state.js]: Array<string>;

Comment (danez) [inline src/tokenizer/state.js]: 👍

**Comment by hzoo** [comment]
cc @dlmr

**Comment by dlmr** [comment]
Looks good! LGTM 👍

File: src/tokenizer/state.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default class State {
     this.containsEsc = this.containsOctal = false;
     this.octalPosition = null;
 
-    this.exportedIdentifiers = {};
+    this.exportedIdentifiers = [];
```
Comment (STRML) [inline src/tokenizer/state.js]: Any reason to not use Object.create(null); here instead?

**Pull request merged**